train.py

```diff
@@ -18,32 +19,33 @@
 from utilities.training_logger import TrainingLogger
 
 
-
-def train(params):
-    """Instantiate and train the HGN.
+def train(params, variational, dtype=torch.float):
+    """Instantiate and train the Hamiltonian Generative Network.
 
     Args:
-        params (dictionary): Experiment parameters (see experiment_params folder)
+        params (dict): Experiment parameters (see experiment_params folder).
+        variational (bool): Whether to perform variational or deterministic training.
```

Comment:
This should not be an extra parameter. Include it in the params dict in the yaml file

Reply:
Yes we will have to, but I thought it would have broken all the things we have running now. Maybe we can check it together this afternoon.

Reply:
I'll do it